server/manager/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1675,9 +1675,8 @@ public boolean delegationTokensAvailable() {
     }
   }
 
-  public void markDeadServerLogsAsClosed(Map<TServerInstance,List<Path>> logsForDeadServers)
-      throws WalMarkerException {
-    WalStateManager mgr = new WalStateManager(getContext());
+  public void markDeadServerLogsAsClosed(WalStateManager mgr,
```

Comment:
I think this method can be static now.

Reply:
`TabletGroupWatcher` was the only thing using this method so I moved it from Master and made it private static.

Reply:
In 73558943033d1a0e0167424e9f06cf834cfb0f39